tests/hardware_tests/test_nav.py

```diff
@@ -0,0 +1,123 @@
+import time
+from copy import deepcopy
+
+import cv2
+import magnum as mn
+import numpy as np
+from spot_rl.envs.gaze_env import construct_config_for_gaze
+from spot_rl.envs.nav_env import construct_config_for_nav
+from spot_rl.envs.place_env import construct_config_for_place
+from spot_rl.envs.skill_manager import SpotSkillManager
+from spot_rl.utils.heuristic_nav import ImageSearch
+from spot_rl.utils.utils import map_user_input_to_boolean
+from spot_wrapper.spot import image_response_to_cv2, scale_depth_img
+
+if __name__ == "__main__":
+    run_the_loop = True  # Don't break the test loop
+    is_in_position = False  # is it before the receptacle
+    # Construct all three configs
+    nav_config, pick_config, place_config = (
+        construct_config_for_nav(),
+        construct_config_for_gaze(),
+        construct_config_for_place(),
+    )
+    # Modify any values in the config
+    # nav_config.SUCCESS_DISTANCE = 0.10
```

Comment:
...
The goal of `test/hardware_tests/` folder is to add all automated hardware CI tests. Please cleanup this file and rename it as `test_heuristic_nav`

Reply:
deleted this file